### ISSUE TITLE
Stochastic Weight Averaging

### ISSUE THREAD

**Issue by jellis18** [nathanhubens/Learning-Rate#1]
Would you be interested in including stochastic weight averaging [(SWA)](https://arxiv.org/pdf/1803.05407.pdf) into the cyclic learning rate callback? If so I can work on this and make a PR. 

I think this package is very useful for Keras users as is but would be even better with SWA.

**Comment by ogencoglu** [comment]
I was about to suggest that. +1

**Comment by mositemp** [comment]
Hi @jellis18 ,
Did you have any progress with the SWA implementation?

**Comment by nathanhubens** [comment]
@jellis18 
Sure great idea ! 
Feel free to make a PR when you are done and I will look into it.

**Comment by jellis18** [comment]
I've started working on this but haven't quite finished it yet. @nathanhubens, do you think you can add an open source license to this software?
